server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+
+app.get('/', (req, res) => {
+
+})
+
+
+
+
+
+const port  = 3000
```

Comment:
treat your README as if it was the documentation for your project.  I should be able to control click on the links on README and get to the response stated.  It has you using port 1337 in documentation.  If you opt to use your own number be sure you are changing it throughout your project.